src/python/nispor/iface.py

```diff
@@ -55,6 +56,8 @@ def _iface_info_to_obj(iface_info):
         iface = NisporBond(iface_info)
     elif iface_type == "Bridge":
         iface = NisporBridge(iface_info)
+    elif iface_type == "Ethernet":
+        iface = NisporSriov(iface_info)
```

Comment:
Please only use `NisporSriov` only when SR-IOV entry found.

Calling all ethenrnet interface as `NisporSriov` is not a good idea.

Reply:
Done, thanks!